extensions/wrapper/wrapper-common-mappers/src/test/java/de/sovity/edc/ext/wrapper/api/common/mappers/AssetMapperTest.java

```diff
@@ -51,7 +52,8 @@ void test_buildAssetDto() {
         assertThat(uiAsset.getParticipantId()).isEqualTo(participantId);
         assertThat(uiAsset.getTitle()).isEqualTo("My Asset");
         assertThat(uiAsset.getLanguage()).isEqualTo("https://w3id.org/idsa/code/EN");
-        assertThat(uiAsset.getDescription()).isEqualTo("Lorem Ipsum ...");
+        assertThat(uiAsset.getDescription()).isEqualTo("# Lorem Ipsum...\n## h2 title\n[Link text Here](example.com)");
```

Comment:
we should also test the abbreviation part. you might make it easier by making this text long enough to be abbreviated, and then adding unit tests just for the "TextUtils#abbreviate", where you can test everything (e.g.  null, empty string, small string, too large string)